assignments/prototype-refactor.js

```diff
@@ -7,3 +7,131 @@ Prototype Refactor
 2. Your goal is to refactor all of this code to use ES6 Classes. The console.log() statements should still return what is expected of them.
 
 */
+
+/*
+  === GameObject ===
+  * createdAt
+  * dimensions
+  * destroy() // prototype method -> returns the string: 'Object was removed from the game.'
+*/
+
+class GameObject{
+    constructor(attributes) {
+        this.createdAt = attributes.createdAt;
+        this.dimensions = attributes.dimensions;
```

Comment:
You could use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here pretty easily:
```javascript
constructor({ createdAt, dimensions }) {
    this.createdAt = createdAt;
    this.dimensions = dimensions;
}
```